spec/beaker/dsl/install_utils/foss_utils_spec.rb

```diff
@@ -396,7 +396,7 @@ def logger
     context 'on el-6' do
       let(:platform) { Beaker::Platform.new('el-6-i386') }
       it 'installs' do
-        expect(subject).to receive(:on).with(hosts[0], /puppetlabs-release-el-6\.noarch\.rpm/)
+        expect(hosts[0]).to receive(:install_package_with_rpm).with('http://yum.puppetlabs.com/puppetlabs-release-el-6.noarch.rpm', '--replacepkgs', {:package_proxy=>false})
```

Comment:
Whether expecting a match on part of the RPM filename or the entire URL, I feel there must be a way of designing these tests where we don't need to make assumptions about URLs or filenames.  However, rspec is a bit of a black art to me so I don't know how to improve upon it.  Happy to make this change as you suggest.